openmdao_book/features/core_features/working_with_groups/configure_method.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>Missing paragraph that follows this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I will add this.